src/converters/fromZigbee.ts

```diff
@@ -4911,10 +4911,20 @@ const converters1 = {
                 // data[1][bit16..bit31]: y
                 // data[0][bit0..bit15] : z
                 // left shift first to preserve sign extension for 'x'
-                const x = ((data['1'] << 16) >> 16);
-                const y = (data['1'] >> 16);
+                let x = ((data['1'] << 16) >> 16);
+                let y = (data['1'] >> 16);
                 // left shift first to preserve sign extension for 'z'
-                const z = ((data['0'] << 16) >> 16);
+                let z = ((data['0'] << 16) >> 16);
+
+                // raw accelrometer values
+                result.raw_x=x;
+                result.raw_y=y;
+                result.raw_z=z;
+
+                // simple offset calibration
+                x=calibrateAndPrecisionRoundOptions(x, options, 'raw_x');
+                y=calibrateAndPrecisionRoundOptions(y, options, 'raw_y');
+                z=calibrateAndPrecisionRoundOptions(z, options, 'raw_z');
```

Comment:
I think it's better to call the options `x` (no `raw_`)

Reply:
oh sorry you were talking about the option name
yes I can rename (the question for the raw values is still valid though)

Reply:
Please rename

Reply:
renamed.
I think It might be confusing as they are offsets for the accelerometer underlying /raw values, not the exposed angle_x y z that are computed...